src/cdk/commands/aslVisualizationCDKManager.ts

```diff
@@ -0,0 +1,94 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+
+import { AslVisualizationCDK } from './aslVisualizationCDK'
+import { ConstructNode } from '../explorer/nodes/constructNode'
+import { getLogger, Logger } from '../../shared/logger'
+import { AbstractAslVisualizationManager } from '../../../src/stepFunctions/commands/visualizeStateMachine/abstractAslVisualizationManager'
+import { StateMachineGraphCache } from '../../../src/stepFunctions/utils'
+
+export class AslVisualizationCDKManager extends AbstractAslVisualizationManager {
+    protected readonly managedVisualizations: Map<string, Map<string, AslVisualizationCDK>> = new Map<string, Map<string, AslVisualizationCDK>>()
+
+    public constructor(extensionContext: vscode.ExtensionContext) {
+        super(extensionContext)
+    }
+
+    public async visualizeStateMachine(
+        globalStorage: vscode.Memento,
+        node: ConstructNode
+    ): Promise<vscode.WebviewPanel | undefined> {
+        const logger: Logger = getLogger()
```

Comment:
Nit: no need to provide a type for a const (it'll infer from the assigned value)